releases/2022.2/2022.2.md

```diff
@@ -0,0 +1,173 @@
+---
+title: 2022.2
+parent: Releases
+---
+
+<div align="center">
+  <h1>Kytos-ng 2022.2 - kiko</h1>
+</div>
+
+<div align="center">
+  <a href="https://www.amlight.net"><img src="https://amlight.net/wp-content/uploads/2016/02/AmLlightExP_s.jpg"></a>
+</div>
+
+Kytos-ng[^1] version `2022.2`, kiko, was released in August.
+
+## Objectives
+
+This version had these major objectives:
+
+- Optimize `mef_eline` EVCs convergence to take place in a few seconds on average.
+- Integrate a production-grade recommended document-oriented database for Napps.
+- Enhance Kytos core to support event message prioritization and have higher performance when handling events in general.
+- Have a better infrastructure tools for application performance monitoring (APM) and logging.
+- New major features such as link liveness detection, interface loop detection, and new Web UI components.
+- Increase unit test coverage to at least 85% on `coloring`, `sdntrace`, and `sdntrace_cp`.
+- Leverage queue loggers to minimize latency.
+
+In order to achieve the objectives of this version, the following enhancements were refined and delivered:
+
+### 1. `mef_eline` EVCs convergence optimizations
+
+TODO: Ask Italo to add and summarize the EVCs convergence gains.
```

Comment:
```suggestion
> Protection in Metro Ethernet Networks is a self-healing property of the network that allows it to continue to function with minimal or no impact on the network users upon disruption, outages, or degradation of links, facilities, or equipment. (MEF 2, Technical Specification)

Aligned with MEF Ethernet Service Protection technical specification, `mef_eline` has been enhanced with strategies to provide fast convergence for fully dynamic EVCs, as documented in  [EP029 blueprint](https://github.com/kytos-ng/kytos/blob/master/docs/blueprints/EP029.rst). In particular, we've introduced an improved convergence time when handling link failure events using maximum disjoint path algorithms and pre-installed failover paths.

`mef_eline` convergence performance is more than 100x faster in handling link failure events for hundreds of EVCs compared to the previous release (see complete performance evaluation and details in [PR #175](https://github.com/kytos-ng/mef_eline/pull/175). In addition, the overall performance achieved a sub-second convergence time (plus control plane latency) for handling a single link failure.
```

Reply:
These two paragraphs look great, very handy to also have a link of PR 175. Thanks @italovalcy, let me commit this.